<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Forvo真人语音下载器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="RC.qrc">
    <normaloff>:/new/prefix1/rc/Applications.ico</normaloff>:/new/prefix1/rc/Applications.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" columnstretch="5,3">
      <item row="0" column="0">
       <widget class="QTableView" name="MainTableView">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>120</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>30</number>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTextBrowser" name="TextBrowser">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="ProgressBar">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_Back"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="action_QT"/>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="action_Transform"/>
    <addaction name="action_StartDownload"/>
    <addaction name="separator"/>
    <addaction name="action_Listen"/>
    <addaction name="separator"/>
    <addaction name="action_DownloadConfig"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Back"/>
   <addaction name="separator"/>
   <addaction name="action_Transform"/>
   <addaction name="action_StartDownload"/>
   <addaction name="separator"/>
   <addaction name="action_Listen"/>
   <addaction name="action_DownloadConfig"/>
   <addaction name="separator"/>
   <addaction name="action_Quit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Back">
   <property name="icon">
    <iconset resource="RC.qrc">
     <normaloff>:/new/prefix1/rc/Home.ico</normaloff>:/new/prefix1/rc/Home.ico</iconset>
   </property>
   <property name="text">
    <string>返回开始</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="RC.qrc">
     <normaloff>:/new/prefix1/rc/Standby.ico</normaloff>:/new/prefix1/rc/Standby.ico</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_QT">
   <property name="icon">
    <iconset resource="RC.qrc">
     <normaloff>:/new/prefix1/rc/qt-logo.png</normaloff>:/new/prefix1/rc/qt-logo.png</iconset>
   </property>
   <property name="text">
    <string>关于QT</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="RC.qrc">
     <normaloff>:/new/prefix1/rc/Applications.ico</normaloff>:/new/prefix1/rc/Applications.ico</iconset>
   </property>
   <property name="text">
    <string>关于软件</string>
   </property>
  </action>
  <action name="action_StartDownload">
   <property name="icon">
    <iconset resource="RC.qrc">
     <normaloff>:/new/prefix1/rc/Download (2).ico</normaloff>:/new/prefix1/rc/Download (2).ico</iconset>
   </property>
   <property name="text">
    <string>开始下载</string>
   </property>
  </action>
  <action name="action_DownloadConfig">
   <property name="icon">
    <iconset resource="RC.qrc">
     <normaloff>:/new/prefix1/rc/Tool2 (2).ico</normaloff>:/new/prefix1/rc/Tool2 (2).ico</iconset>
   </property>
   <property name="text">
    <string>下载配置</string>
   </property>
  </action>
  <action name="action_Transform">
   <property name="icon">
    <iconset resource="RC.qrc">
     <normaloff>:/new/prefix1/rc/Wait.ico</normaloff>:/new/prefix1/rc/Wait.ico</iconset>
   </property>
   <property name="text">
    <string>文件转换</string>
   </property>
  </action>
  <action name="action_Listen">
   <property name="icon">
    <iconset resource="RC.qrc">
     <normaloff>:/new/prefix1/rc/Player Play (2).ico</normaloff>:/new/prefix1/rc/Player Play (2).ico</iconset>
   </property>
   <property name="text">
    <string>发音试听</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RC.qrc"/>
 </resources>
 <connections/>
</ui>
